### PR TITLE
Update kidflashwallywest.ms

### PR DESCRIPTION
**Pull request by MrCraft888**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/kidflashwallywest.ms [hunk 1]
```diff
@@ -6,6 +6,12 @@
...
+ +                     'sig': 'vLm5HtbiO4tLQAcW2DIiKd22lnE8B9KePfqe/kvoFJU7eRyBrczhMdbd+S+e3VfM/ffXYcyM6HuBd/IYVqqXwfGtmGq88O6lQMs/I2B/C0bDZEzs3nFMH9W47oBt4N9B0RfOJ6UgIfle9OQvK06Twg/OGOSKnZ7aei9WPE2/rxXvZ5d68dQ2j3kDQoGEHLbnPuynYo7UKLfx/z8OOi0zKwGEFv8ZJQoZcXgqZNIWeCHXZXADQ5fiF/qXAv5zElduiUkMHckqjGSrM1fHr8XHh6HKvXX9mjD4hpxf6gFMiQdW9b4Z1S67xSYGCtzFFvs4gJk/KeOLkskRBCC2kPTKtyB+jLVSn7/cng968KOzC2AySLukFfasFNi4udZIHC85Naygf6/kSfkuWUYxn6Mj91K5DP0QPpoayYj2GeroleyKW77laJwN4uEUU9m6z0zY3VLyl0UzQLwYWBIyAGFZE75y5qTJeVzNI6fWP/cx0Pl44Bcb5PNm0yF8naFwwJhYoClNCqJYjrTjkGVuO1wTG3yaOYnYV7/0JfFWNdiu8vWJAbObssW3fgZq+k6pF8Pz29NYdpP9TW3WEQ3O1cvnfU98LvYIop/hGZxN3+mSySWbLTU/EBBJe365cw9CuoA8juDzZiKbSuth7rIR0bQkkeB9TvYOsxQpS5l8WzkQ4Y0=',                    
+ +                     'val': 'eyJ0aW1lc3RhbXAiOjE0ODg0MDEyOTUzNDMsInByb2ZpbGVJZCI6IjQ3NDcwN2Y3ZjhiNDQzZWE4OTc4OWNhZTE3OTE3ZTgyIiwicHJvZmlsZU5hbWUiOiJNaWxlc01vcmFsZXM4ODgiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzg0NTQ0YThhM2E5MzFiZjliMzZjY2ZjYzRmNGRhYmNjMTdkNmExZGU0MzQ3YWU3NTg1ZGRmZDZhNTUzYTUxZSJ9fX0='              
+ +                )
+ +            ),
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/kidflashwallywest.ms]: remove the plus signs

**Comment by malonnnn** [comment]
link to the skin you're updating with

**Comment by MrCraft888** [comment]
http://www.planetminecraft.com/skin/wally-west-3819254/

File: CommandHelper/LocalPackages/commands/buffs/kidflashwallywest.ms [hunk 2]
```diff
@@ -6,6 +6,12 @@
...
+ +                    'sig': 'Rbn7c6K4Qm5GKEFCiS0s+MlAaksff1RT5DLsKfBba+Iw/UliMpMlM0XQ6KfiCUYvnXkUpIiTqPIUxl7lhOF/kcebcyIfuFKR10WdHgOvaD3bJf3A7yhtjb/Dnf2VKQJ3HGC8Fnj2WjAIgIfy1dlCXVOkXQNcsPH+2KYsT5unZsJRaBW8FpVSbG9FBojajq5KN0neQbxsVENZj+N7w/lU8pPt4nQTT/mtpvsRu8woaCXvCP7ELKePDwTTGwD8DpJSlSuPgXyGQznNW/HlCA47jzW6/gv+cb91g3fa0YBleA6e9X8y+pcp/I5J0d+nmTP/05Z4o7Xe+PH6K+o3m7+75Zn5VtiX7Dx2MA5iuGU8M0b1dshgQBH/b4wLK2ofkl4a6sS3/XF+ggfkprTUUmMJM5bX201AV7w5Yrg6d+n8FtJIXqViZxPZ4sme5HBeZhmnHSWliSYtDYNtwgEB00/RMHzwCCwh6nnj/FYqTmU1af9oj/SZsefr30LXnd+CkzoU8LfgSjsrwSSr8D33NcC3ubjXKoDOxjn7pge0p3t/fs+9kG0B3Zzpc00VsJpFU/3r1GgoTot+JnluyA32t4aukuBAhsyEvzyO7sK93FQ0FdtZGqfSzTeK5Q9CgPDwSihd5JAV/v2peeWK0sYVepK509YDfmM9MBrIuD9sWFUnmEg=',                    
+ +                    'val': 'eyJ0aW1lc3RhbXAiOjE0ODgwNjQ3MzkyMzMsInByb2ZpbGVJZCI6IjQ3NDcwN2Y3ZjhiNDQzZWE4OTc4OWNhZTE3OTE3ZTgyIiwicHJvZmlsZU5hbWUiOiJNaWxlc01vcmFsZXM4ODgiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzc5ZTYxNmJkM2VhYTNmOTE4ZWEyZTdiZDZhN2QzYTdkZWE3MWViMWI5MzIzZGJjMTY2NTc3YjIxZDM2MzA1NSJ9fX0='                
+ +                )
+ +            ),
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/kidflashwallywest.ms]: remove the plus signs

**Pull request closed without merging**